tracer/src/Datadog.Trace.BenchmarkDotNet/DatadogColumnHidingRule.cs

```diff
@@ -1,30 +0,0 @@
-// <copyright file="DatadogColumnHidingRule.cs" company="Datadog">
-// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
-// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
-// </copyright>
-
-using BenchmarkDotNet.Columns;
-
-namespace Datadog.Trace.BenchmarkDotNet;
-
-/// <summary>
-/// Datadog column hiding rule
-/// </summary>
-public class DatadogColumnHidingRule : IColumnHidingRule
```

Comment:
You can't delete this, this is a breaking change

Reply:
are you sure 🤣 ? 💣 💣 💣